packages/core/theme-module/theme/pages/MyAccount/MyProfile.vue

```diff
@@ -144,14 +150,29 @@ export default {
       default: () => ({})
     }
   },
+  setup() {
+    const { user, changePassword, error } = useUser();
+    const form = ref({});
+
+    const updatePassword = async () => {
```

Comment:
what about this approach?
```TS
const updatePassword = () => changePassword(form.value.currentPassword, form.value.newPassword)
  .then(() => {
    if (!error.value) {
      form.value = {};
    }
  })
```

Reply:
Well, it's the same but much more nested and thus less readable, in my opinion.